ChiModules/LinearBoltzmannSolver/IterativeOperations/lbs_setsource.cc

```diff
@@ -140,29 +142,9 @@ SetSource(LBSGroupset& groupset,
             for (size_t gprime = first_grp; gprime <= last_grp; ++gprime)
             {
               if ((gprime < gs_i) or (gprime > gs_f))
-              {
-                //without delayed neutron precursors
-                if (not options.use_precursors)
-                  infission_g += xs->chi[g] *
-                                 xs->nu_sigma_f[gprime] *
-                                 phi_old_local[ir + gprime];
-
-                //with delayed neutron precursors
-                else
-                {
-                  //==================== Prompt fission
-                  infission_g += xs->chi_prompt[g] *
-                                 xs->nu_prompt_sigma_f[gprime] *
-                                 phi_old_local[ir + gprime];
-
-                  //==================== Delayed fission
-                  for (size_t j = 0; j < xs->num_precursors; ++j)
-                    infission_g += xs->chi_delayed[g][j] *
-                                   xs->precursor_yield[j] *
-                                   xs->nu_delayed_sigma_f[gprime] *
-                                   phi_old_local[ir + gprime];
-                }
-              }
+                infission_g += xs->chi[g] *
+                               xs->nu_sigma_f[gprime] *
+                               phi_old_local[uk_map + gprime];
```

Comment:
Good catch. Fixed

Reply:
Okay time out, #179 addressed #150. Are we planning on reverting this now?

Reply:
@zachhardy We should not be reverting so please help me determine if the rebase is consistent now. Please check the raw file since this comment-stream is now based on an outdated diff. The only major change in this routine should be replacing
```c++
if (apply_mat_src && (ell == 0))
  destination_q[ir + g] += src[g];
```
with
```c++
if ( apply_mat_src and (ell == 0) and (not options.use_src_moments))
  destination_q[uk_map + g] += src[g];
else if (apply_mat_src and options.use_src_moments)
  destination_q[uk_map + g] += ext_src_moments_local[uk_map + g];
```
and of course the renaming of `ir` to `uk_map`.